src/components/HomePageComponent.jsx

```diff
@@ -61,37 +56,55 @@ class HomePageComponent extends React.Component {
-                          && <div className="img-wrapper" style={{ background: `url('${article.image_url}') center center` }}>&nbsp;</div>
-                        }
-                        <a href={`/article/${article.slug}`}>{article.title}</a>
-                        <p style={{ margin: 0 }}>
-                          <i className="grey-text">
-                            By
-                            {' '}
-                            {article.author.username}
-                          </i>
-                        </p>
-                        <p>{article.description}</p>
-                        {article.tags
-                            && article.tags.map(tag => <Chip key={tag}>{tag}</Chip>)
+                        <div
+                          className="img-wrapper"
+                          style={{ background: `url('${article.image_url ? article.image_url : config.PLACEHOLDER_IMAGE}') center center` }}
+                        >
+                          &nbsp;
+                        </div>
+                        <div>
+                          <a href={`/article/${article.slug}`}>{article.title}</a>
+                          <p>
+                            <i className="grey-text">
+                              By
+                              {' '}
+                              {article.author.username}
+                            </i>
+                          </p>
+                          <p>{article.description}</p>
+                          {article.tags
+                              && article.tags.map(tag => <Chip key={tag}>{tag}</Chip>)
                           }
+                        </div>
                       </div>
                     ))}
+                    {articles.count > config.PAGE_SIZE
+                      && (
+                      <Pagination
+                        onSelect={this.onPageChange}
+                        items={Math.ceil(articles.count / config.PAGE_SIZE)}
+                        activePage={1}
+                        maxButtons={5}
+                      />
+                      )
+                    }
                   </Col>
                   <Col m={3} s={12} className="featured">
                     <h5 className="header">Recent Articles</h5>
-                    {recentArticles.map(article => (
+                    {articles.recent.map(article => (
                       <div key={article.slug} className="article card">
                         <a href={`/article/${article.slug}`}>
-                          {article.image_url
-                            && <div className="img-wrapper" style={{ background: `url('${article.image_url}') center center` }}>&nbsp;</div>
-                          }
+                          <div
+                            className="img-wrapper"
+                            style={{ background: `url('${article.image_url ? article.image_url : config.PLACEHOLDER_IMAGE}') center center` }}
```

Comment:
Can we have this as a class instead of an inline style

Reply:
Hello, about the inlined css, in this case, it is an exception because I am using the image from the article to provide a dynamic background for the `div`.

The reason I am using a `div` instead of just an image, is that it is a styling hack that allows images to retain their aspect ratio.

Otherwise, the images may appear skewed or may not retain a uniform size.